### ISSUE TITLE
How to disable scrolling when pressing spacebar?

### ISSUE THREAD

**Issue by davinjaya98** [idiotWu/smooth-scrollbar#205]
Hi I am doing a page with an editor in it. The editor that I'm using is summernote editor. The editor works by changing a textarea into a div shaped rich text editor. This is where the problem kicks in. 

Because the rich text editor is in div format, everytime spacebar is pressed, smooth scrollbar will trigger the scroll down effect. 

Is there anyway to disable the spacebar scrolling login on smooth scrollbar?

**Comment by idiotWu** [comment]
Inside your editor, try:

```js
editor.addEventListener('keydown', (event) => {
    event.preventDefault();
    // or
    // event.stopPropagation();
});
```

**Comment by idiotWu** [comment]
Spacebar scrolling is fixed via #210. Please update your scrollbar to v8.4.1 :)

**Comment by davinjaya98** [comment]
Thanks for the update. sorry I didn't follow up earlier!
